<launch>
	<arg name="model" default='$(find beginner_tutorials)/urdf/ruchomy.urdf' />
	<arg name="gui" default="True" />
	<param name="robot_description" textfile="$(arg model)" />
	<param name="link1" type="double" value="1.0" />
	<param name="arm2" type="double" value="0.5" />
	<param name="joint2_lower" type="double" value="-3.915" />
	<param name="joint2_upper" type="double" value="0.775" />
	<param name="joint3_lower" type="double" value="-2.365" />
	<param name="joint3_upper" type="double" value="2.365" />
	<param name="use_gui" value="$(arg gui)"/>
	<node pkg="beginner_tutorials" name="DKIN" type="DKIN"/>
	<node pkg="beginner_tutorials" name="KDLDKIN" type="KDLDKIN"/>
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find beginner_tutorials)/urdf.rviz" required="true" />
</launch>
